include/zck.h.in

```diff
@@ -85,7 +86,8 @@ ssize_t zck_write(zckCtx *zck, const char *src, const size_t src_size)
 /* Create a chunk boundary */
 ssize_t zck_end_chunk(zckCtx *zck)
     __attribute__ ((warn_unused_result));
-
+/* Create the database for uthash if not present (done automatically by read */
+bool zck_create_hashdb(zckCtx *zck);
```

Comment:
Let's rename this to `zck_generate_hashdb`

Reply:
Ok